include/cgimap/api06/changeset_upload/way.hpp

```diff
@@ -24,15 +25,15 @@ class Way : public OSMObject {
     osm_nwr_signed_id_t _waynode = 0;
 
     try {
-	_waynode = std::stol(waynode);
+      _waynode = std::stol(waynode);
     } catch (std::invalid_argument& e) {
-	throw xml_error("Way node is not numeric");
+      throw xml_error("Way node is not numeric");
     } catch (std::out_of_range& e) {
-	throw xml_error("Way node value is too large");
+      throw xml_error("Way node value is too large");
```

Comment:
In a few of the files there were a mixture of tabs and spaces that were fixed during formatting.